src/registry/views/static/component-info.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
```

Comment:
Sure. It's strange because there were some client files that crashed setting `const` and `let` due to the `use strict`, but not these two files 😕